telegram-bot-api/Client.cpp

```diff
@@ -7693,7 +7698,20 @@ void Client::process_2fapassword_query(PromisedQueryPtr &query) {
   }
   send_request(make_object<td_api::checkAuthenticationPassword>(password.str()),
                std::make_unique<TdOnAuthorizationQueryCallback>(this, std::move(query)));
+}
+
+void Client::process_register_user_query(PromisedQueryPtr &query) {
+  auto first_name = query->arg("first_name");
+  if (first_name.empty()) {
+    return fail_query(400, "Bad Request: first_name not found", std::move(query));
+  }
+  auto last_name = query->arg("last_name");
```

Comment:
last_name should never be empty, rather put it as null, because telegrams block the signup under certain circumstances if you enter an empty string as last_name

Reply:
false, it must not be null, otherwise it will ban the user!

Reply:
An empty last name must be an empty string